examples/gke/gke.sh

```diff
@@ -79,9 +79,12 @@ then
   exit 1
 fi
 
-GCP_REGION=${GCP_ZONE:0:$((${#GCP_ZONE}-2))}
+gcloud() {
+  command gcloud "$@" --project "${GCP_PROJECT}" --zone "${GCP_ZONE}"
```

Comment:
nit: I don't think gcloud is that sensitive but I'd put the `"$@"` at the end, I know some programs (although unfortunate) require flags before args

Reply:
guess that can go the other way too so let's keep what's working ;)

Reply:
exactly:
```
++ gcloud --project skilled-adapter-452 --zone us-east1-b container get-server-config --format 'value(validMasterVersions[0])'                                                                                  
ERROR: (gcloud) Invalid choice: 'us-east1-b'.                                                                                                                                                                                                                                                                                                                                                                                    
Maybe you meant:
  gcloud config get-value
  gcloud config list
  gcloud config set
  gcloud config unset
  gcloud container get-server-config

To search the help text of gcloud commands, run:
  gcloud help -- SEARCH_TERMS